internal/exec/selected/selected.go

```diff
@@ -234,6 +265,7 @@ func applyField(r *Request, s *resolvable.Schema, e resolvable.Resolvable, sels
 	}
 }
 
+// FIXME: move these to directive visitors?
```

Comment:
These are applied on `FIELD`, `INLINE_FRAGMENT` and `FRAGMENT_SPREAD` so they are executable directives (applied by the user on the query fields). Therefore, they are properties of the request and not part of the schema directives. I wouldn't worry about them for now. Also, every GraphQL server should implement them - do you want to pass them as Schema option? I'd leave these for now.